### DISCUSSION TITLE
Set open/close color popup when arriving by TAB key

### DISCUSSION THREAD

**Discussion by Ale-Farneda** [viclafouch/mui-color-input#27] [General]
I'm having trouble trying to control the open/close popup on focus/blur/focusCapture...
Could the popup receive an open close prop so i can close it after select and open it on focus the text input.

**Comment by umarkhan04** [comment]
This need to be addressed "I'm having trouble trying to control the open/close popup on focus/blur/focusCapture...
Could the popup receive an open close prop so i can close it after select and open it on focus the text input."
without it is pretty much useless user should have control over it
